.github/workflows/ci.yml

```diff
@@ -194,6 +201,11 @@ jobs:
         if: (matrix.device_type == 'FPGA_EMU')
         run: |
           sudo apt-get install intel-oneapi-compiler-fpga -y
+      - name: Install libfmt and gtest needed by Distributed Ranges tests
+        if: (matrix.std == '23')
+        run: |
+          sudo apt-get install libgtest-dev libfmt-dev -y
+          sudo apt-get remove libgcc-14-dev cpp-14-x86-64-linux-gnu -y  # gcc14 toolkit not supported yet by DR
```

Comment:
We've fixed gcc14 support in https://github.com/oneapi-src/oneDPL/pull/1721 and I've removed this line. The line was that because icpx selected the newest existing gcc toolkit.